tsconfig.json

```
@@ -4,11 +4,11 @@
     "esModuleInterop": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "noEmit": true,
+    "outDir": "lib",
     "resolveJsonModule": true,
     "skipLibCheck": true,
     "strict": true,
     "target": "es2020",
     "typeRoots": ["node_modules/@types", "typings"]
-  }
+  },
```

```suggestion
  }
```

I think JSON doesn't allow trailing commas (although `tsc` seams to accept it).